src/content/docs/style-guide/writing-docs/article-templates/branching-docs-guide.mdx

```diff
@@ -0,0 +1,254 @@
+
+## New components [#new-components]
+
+The branching doc introduces some new components. Some of these components are available for use across the docs site, whether it's a branching doc or not.
+
+The new universal components are:
+
+* installation-specific feedback form
+* tab switcher
+
+The branching-specific components are:
+
+* dropdown menus with image-based tile selection
+* numbered steps
+* embedded, customizable config (or text) file that can be copied or downloaded
+
+### Installation feedback form [#installation-feedback]
+
+Stay tuned!
+
+### Tab switcher [#tab]
+
+The tab switcher, like collapsers, is a way of presenting sets of content in a single location.
+
+```
+<Tabs>
+	<TabsBar>
+		<TabsBarItem id=""></TabsBarItem>
+		<TabsBarItem id=""></TabsBarItem>
+		<TabsBarItem id=""></TabsBarItem>
+	</TabsBar>
+
+	<TabsPages>
+		<TabsPageItem id="">
+		</TabsPageItem>
+		<TabsPageItem id="">
+		</TabsPageItem>
+		<TabsPageItem id="">
+		</TabsPageItem>
+	</TabsPages>
+</Tabs>
+```
+
+
+### Tile dropdown [#tile-dropdown] 
+
+The tile dropdown is how a reader controls the branching content that's shown on a doc. The dropdown content and structure is controlled via the branching doc's YAML file.
+
+### Numbered steps [#numbered-steps]
+
+Nearly every MDX file in a branching doc is represented by a numbered step. The branching doc's numbered steps are automatically generated via the YAML file.
+
+### Embedded config file [#config]
+
+The config file is added to the `/install/config/agent-config` directory. Specific lines can be targeted in the config file for update by the reader.
+
+## How it works [#how]
+
+The install docs live in the `install` directory. There is a single `.yaml` file that defines how and when specific content is swapped in and out of the doc. The specific pieces of content are represented as `.mdx` files.
+
+## Directory overview [#install-directory]
+
+The install framework content lives in `src/install`. 
+
+To create a new install doc, add a directory to `install`.
+
+Your directory structure needs to look like this:
+
+```
+install
+|--assets (used for install images)
+|--config (used for the install yaml files)
+   |--agent-config (used for the agent config yml files)
+|--specific-install (used for a specific install doc)
+   |--sub-directory1 (used for .mdx files that will be swapped in)
+   |--sub-directory2 (used for .mdx files that will be swapped in)
+   |--sub-directoryx (you can have as many sub-directories as you need)
+```
+
+## The branching YAML file [#yaml-overview]
+
+The config (yaml) file is a highly customizable way to generate a wide variety of custom, interactive content. Being a YAML file, it’s extremely important to pay attention to indents and to make sure that every piece of the YAML file is correct. 
```

Comment:
@paperclypse it looks like this file needs the four-letter suffix, right? If so, could you call that out in a tip or warning?